lib/cocoapods/target.rb

```diff
@@ -180,6 +180,12 @@ def dummy_source_path
       support_files_dir + "#{label}-dummy.m"
     end
 
+    # @return [String] The version associated with this target
+    #
+    def version
+      '1.0.0'
```

Comment:
NIT: Seems like maybe this should be an attr_reader initialized on creating the target, instead of a method with a hardcoded value?

Reply:
Or make this a constant at the top and call it `DEFAULT_VERSION` or something?

Reply:
will make it a const